libopae/plugins/xfpga/reconf.c

```diff
@@ -400,8 +404,8 @@ fpga_result __FPGA_API__ xfpga_fpgaReconfigureSlot(fpga_handle fpga,
 		}
 
 		// Set power threshold for integrated fpga.
-		if (hw_type == FPGA_HW_MCP) {
-
+		if (hw_type == FPGA_HW_MCP &&
+			!stat(FPGA_SYSFS_CLASS_PATH_INTERNAL, &st)) {
```

Comment:
Isn't it an error if the file doesn't exist?

Reply:
Upstream driver doesn't support  set threshold for MCP, only internal driver supports it.

enum fails if  path doesn't exists FPGA_SYSFS_CLASS_PATH_INTERNAL